modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_0_0/PollsUpgradeProcess.java

```diff
@@ -0,0 +1,1074 @@
+
+	private Map<Long, String> _getDDMFormFieldOptionsValues(
+			DDMFormFieldOptions ddmFormFieldOptions, long questionId)
+		throws Exception {
+
+		Map<Long, String> ddmFormFieldOptionsValues = new LinkedHashMap<>();
+
+		try (PreparedStatement preparedStatement = connection.prepareStatement(
+				"select choiceId, description, name from PollsChoice where " +
+					"questionId = ? order by name")) {
+
+			preparedStatement.setLong(1, questionId);
+
+			try (ResultSet resultSet = preparedStatement.executeQuery()) {
+				while (resultSet.next()) {
+					String optionValue = getDDMFormFieldName(
+						LanguageUtil.get(_getResourceBundle(), "option"));
+
+					ddmFormFieldOptionsValues.put(
+						resultSet.getLong("choiceId"), optionValue);
+
+					createDDDMFormFieldOptions(
+						ddmFormFieldOptions, resultSet.getString("description"),
+						resultSet.getString("name"), optionValue);
+				}
+			}
+		}
+
+		return ddmFormFieldOptionsValues;
+	}
+
+	private Map<String, Long> _getDDMFormInstanceResourceActions() {
+		List<ResourceAction> resourceActions =
+			_resourceActionLocalService.getResourceActions(
+				_CLASS_NAME_DDM_FORM_INSTANCE);
+
+		Stream<ResourceAction> stream = resourceActions.stream();
+
+		return stream.collect(
+			Collectors.toMap(
+				resourceAction -> resourceAction.getActionId(),
+				resourceAction -> resourceAction.getBitwiseValue()));
+	}
+
+	private DDMFormLayoutPage _getDDMFormLayoutPage(DDMFormField ddmFormField) {
+		DDMFormLayoutPage ddmFormLayoutPage = new DDMFormLayoutPage();
+
+		ddmFormLayoutPage.addDDMFormLayoutRow(
+			_getDDMFormLayoutRow(ddmFormField));
+		ddmFormLayoutPage.setTitle(
+			_getLocalizedValue(_defaultLocale, StringPool.BLANK));
+
+		return ddmFormLayoutPage;
+	}
+
+	private DDMFormLayoutRow _getDDMFormLayoutRow(DDMFormField ddmFormField) {
+		DDMFormLayoutRow ddmFormLayoutRow = new DDMFormLayoutRow();
+
+		ddmFormLayoutRow.addDDMFormLayoutColumn(
+			new DDMFormLayoutColumn(
+				DDMFormLayoutColumn.FULL, ddmFormField.getName()));
+
+		return ddmFormLayoutRow;
+	}
+
+	private LocalizedValue _getLocalizedValue(Locale locale, String value) {
+		LocalizedValue localizedValue = new LocalizedValue(locale);
+
+		localizedValue.addString(locale, value);
+
+		return localizedValue;
+	}
+
+	private String _getName(String title) {
+		Map<Locale, String> localizations = new HashMap<>();
+
+		for (Locale availableLocale : _availableLocales) {
+			localizations.put(
+				availableLocale,
+				LocalizationUtil.getLocalization(
+					title, LocaleUtil.toLanguageId(availableLocale)));
+		}
+
+		return LocalizationUtil.updateLocalization(
+			localizations, StringPool.BLANK, "Name",
+			LocaleUtil.toLanguageId(_defaultLocale));
+	}
+
+	private String _getOptionLabel(
+		Locale availableLocale, String description, String name) {
+
+		return StringBundler.concat(
+			name, StringPool.PERIOD, StringPool.SPACE,
+			LocalizationUtil.getLocalization(
+				description, LocaleUtil.toLanguageId(availableLocale)));
```

Comment:
Sure @natocesarrego!

The `PollsChoice` table has two columns to save the description of each choice:

1. The `name` column is used to indicate the position of each choice. The possible values ​​can be "a", "b", "c"...
2. The `description` column, as the name already says, is the description of each choice.

<img width="1222" alt="Captura de Tela 2021-08-31 às 15 18 18" src="https://user-images.githubusercontent.com/34186632/131555454-c9229a8f-9df0-4c3e-854c-326fe8558ed6.png">

So it is necessary to concatenate the "name" with the "description" so that the final result (in the SingleSelection field) looks like:

Reply:
Thanks for your clarification, @carolmariaabb! :smile: